doc/sphinx/source/esmvalcore/recipe.inc

```diff
@@ -1,5 +1,97 @@
 .. _recipe:
 
-******
-Recipe
-******
+*****************
+ESMValTool recipe
+*****************
+
+Recipes are the instructions telling ESMValTool
+about the user who wrote the recipe,
+the datasets which need to be run,
+the preprocessors that need to be applied,
+and the diagnostics which need to be run over the preprocessed data.
+This information is provided to ESMValTool in the recipe sections:
+`Documentation`_, `Datasets`_, `Preprocessors`_ and `Diagnostics`_, respectively.
+
+
+Documentation
+=============
+
+The documentation section includes:
+
+- The recipe's author's user name
+- A description of the recipe
+- The user name of the maintainer
+- A list of scientific references
+- the project or projects associated with the recipe.
+
+Note that the author, project, and references will need to be included in the
+config-references.yml file. The author name uses the format:
+last name (clipped after 4 letters) underscore first name (clipped to two letters)
+and all letters are lowercase. 
+For instance, Mickey Mouse would be: mous_mi.
+This is unlikely to be the same as the github user name.
+
+Datasets
+========
+
+The datasets section includes:
+
+- Which Models to run
+- Project (CMIP5 or 6, observations...)
+- Scenario (historicalk/ RCP8.5 etc...)
```

Comment:
This is actually `experiment` (also historicalk --> historical)

Reply:
```suggestion
- experiment (historical/ RCP8.5 etc...)
```